src/modules/firewall/variables.tf

```diff
@@ -49,3 +49,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "firewall_disable_snat" {
+  description = "The address space to be used to ensure that SNAT is disabled."
+  default     = ["0.0.0.0/0"]
+  type        = list
+}
```

Comment:
Looks like this should work functionally! Only feedback is that it might make the intent clearer to other readers named something like `disable_snat_ip_range` and perhaps provide some helpful documentation about why we do this:

```suggestion
# To configure Azure Firewall to never SNAT regardless of the destination IP address, 
# use 0.0.0.0/0 as your private IP address range. 
# With this configuration, Azure Firewall can never route traffic directly to the Internet.
# see: https://docs.microsoft.com/en-us/azure/firewall/snat-private-range
variable "disable_snat_ip_range" {
  description = "The address space to be used to ensure that SNAT is disabled."
  default     = ["0.0.0.0/0"]
  type        = list
}
```

Reply:
Thank you Glenn, great suggestion. I'll modify the code to reflect these changes.